inc/sainexthopgroup.h

```diff
@@ -219,6 +255,33 @@ typedef enum _sai_next_hop_group_member_attr_t
      */
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_MONITORED_OBJECT,
 
+    /**
+     * @brief Object index in the fine grain ECMP table.
+     *
+     * Index specifying the strict member's order.
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP.
+     * Allowed value range for is from 0 to SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE - 1.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_ONLY
+     * @default 0
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_ATTR_INDEX,
+
+    /**
+     * @brief Object priority for enforcing the members' order.
+     *
+     * Index specifying the member's order. The index is not strict exhibiting the behavior of
+     * the priority. Members don't have to have the sequential priorities and it's driver's job
+     * to translate the priorities to the real indeces in the group.
```

Comment:
The plural is indices and not indeces, and then perhaps the word won't be needed to be added to the dictionary

Reply:
Done